app/controllers/reviews_controller.rb

```diff
@@ -31,24 +31,25 @@ def create
     @reviews = @article.reviews
     @review.article_id = @article.id
     @review.user_id = @user.id
-    
+
     if @review.save
       redirect_to article_path(@article), :notice => "Review successfully added"
     else
       redirect_to article_path(@article), :notice => "Review did fail"
-    
+
     end
   end
 
   def destroy
-		@review = Review.find(params[:id])
+    @review = Review.find(params[:id])
     @article = @review.article
 
-		if @review.destroy
-			redirect_to article_path(@article), notice: "Review deleted"
-		else
-			render article_path(@article), notice: "Review was not deleted"
-		end
+    if @review.destroy
+      redirect_to article_path(@article), notice: "Review deleted"
+    else
+      flash[:notice] = "Review was not deleted"
+      render 'show'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.